neptune/projects.py

```diff
@@ -388,6 +395,12 @@ def create_experiment(self,
             notebook_id=notebook_id
         )
 
+        if notebook_id is not None and notebook_path is not None:
+            create_checkpoint(backend=self._backend,
```

Comment:
I just noticed this, but it should be the other way around - it's the experiment that should get the checkpoint_id, not checkpoint getting experiment_id.

I'd treat it similarly to source files.